crates/sui-graphql-rpc/src/types/checkpoint.rs

```diff
@@ -37,6 +41,12 @@ pub(crate) struct CheckpointId {
     pub sequence_number: Option<UInt53>,
 }
 
+/// `DataLoader` key for fetching `StoredRawCheckpoint` by its sequence number.
+#[derive(Copy, Clone, Hash, Eq, PartialEq, Debug)]
+struct SeqNum {
```

Comment:
nit (naming): We already have a `SeqNumKey` below, so maybe it's worth clarifying that this is the `RawSeqNumKey`?